test/e2e/framework/meta.go

```diff
@@ -7,14 +7,16 @@ import (
 )
 
 const (
-	SerialLabelName            = "Serial"
-	RequiresClusterIPLabelName = "RequiresClusterIP"
+	SerialLabelName                = "Serial"
+	RequiresObjectStorageLabelName = "RequiresObjectStorage"
```

Comment:
nit: the vars are ordered the other way around

Reply:
done